pom.xml

```diff
@@ -343,6 +343,17 @@
             <version>2.1</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
+            <version>3.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
@gumbelmj May be it is not a good idea to add a new mocking framework here. You should ask the architect. I suggest to remove the dependency and remove testing private methods.